### PR TITLE
User Login Controller

### PR DESCRIPTION
**Pull request by tylarpierson**
# Description

Login Controller to make sure a user is logged in to leave a review.

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [x] Work-in-Progress, PR is for discussion/feedback

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [x] There are no merge conflicts
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by tylarpierson** [comment]
[Create Business Login Controller](https://trello.com/c/gUAa36yL/32-create-business-login-controller)

**Comment by tylarpierson** [comment]
[Create Business Login Controller](https://trello.com/c/gUAa36yL/32-create-business-login-controller)

**Comment by brad-mortensen** [comment]
[Create Business Login Controller](https://trello.com/c/gUAa36yL/32-create-business-login-controller)

File: app/controllers/api/v1/review_controller.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def create
 
         # current 
         @review = Review.new(text: params[:value], business_id: @business.id, user_id: session[:user_id])
-        puts @review.user.username, "ohhh loard please"
+        # puts @review.user.username, "ohhh loard please"
```
Comment (bpleslie) [inline app/controllers/api/v1/review_controller.rb]: Probably want to just delete this line 😄

Comment (tylarpierson) [reply app/controllers/api/v1/review_controller.rb]: Deleted it

**Pull request merged**